vllm/model_executor/models/llava.py

```diff
@@ -438,7 +438,7 @@ def _get_layer_index(feature_layer_index: int, num_hidden_layers: int) -> int:
     """
     if feature_layer_index < 0:
         return num_hidden_layers + feature_layer_index + 1
-    return feature_layer_index + 1
```

Comment:
This is the cause for the extra block being loaded, e.g., given `num_hidden_layers=24`, `feature_layer_index=-1` produces `24`, but `feature_layer_index=24` produces `25`